ci-operator/step-registry/odo/tests/README.md

```diff
@@ -0,0 +1,37 @@
+# odo-tests-ref<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+
+- [Purpose](#purpose)
+- [Process](#process)
+- [Prerequisite(s)](#prerequisites)
+  - [Infrastructure](#infrastructure)
+  - [Environment Variables](#environment-variables)
+  - [Other](#other)
+- [Custom Image - `oc-bin-image`](#custom-image---oc-bin-image)
+
```

Comment:
Maybe include a General info section here to make the source of the test code easy to navigate to.

For example: [General Information](https://github.com/openshift/release/tree/master/ci-operator/config/windup/windup-ui-tests#general-information)

Reply:
Added General Section